src/oracle/connection/stmt.rs

```diff
@@ -17,13 +17,20 @@ pub struct Statement {
     buffers: Vec<Box<[u8]>>,
     sizes: Vec<i32>,
     indicators: Vec<Box<ffi::OCIInd>>,
+    pub mysql: String,
 }
 
 const NUM_ELEMENTS: usize = 20;
 
 impl Statement {
     pub fn prepare(raw_connection: &Rc<RawConnection>, sql: &str) -> QueryResult<Self> {
-        let mysql = sql.to_string();
+        let mut mysql = sql.to_string();
+        // TODO: this can go wrong: `UPDATE table SET k='LIMIT';`
+        if let Some(pos) = mysql.find("LIMIT") {
```

Comment:
We should have an open issue for this, so we have at least some sort of documentation what may be broken and what should be fixed as soon as certain language features arrive.